file_name_grasp_total: "/AtestCollisionAvoidance/D_050510/model_40/box_db"
file_name_grasp_svm: "/AtestCollisionAvoidance/D_050510/model_40/box_estimate"